src/com/intellij/idea/plugin/hybris/properties/impl/PropertiesServiceImpl.kt

```diff
@@ -142,52 +104,114 @@ class PropertiesServiceImpl(val project: Project) : PropertiesService {
                     "env.properties" -> envPropsFile = file
```

Comment:
these 4 constants can be declared in the `HybrisConstants`, we even have few of them.